js/src/run_trees.ts

```diff
@@ -733,6 +734,9 @@ export class RunTree implements BaseRun {
   }
 
   async postRun(excludeChildRuns = true): Promise<void> {
+    if (this._awaitInputsOnPost) {
```

Comment:
See comment below in `traceable`

Reply:
_nit_: Worth adding the comment inline here?